src/terraform/devcontainer-feature.json

```
@@ -47,6 +47,11 @@
             "type": "boolean",
             "default": false,
             "description": "Install terraform-docs, a utility to generate documentation from Terraform modules"
+        },
+        "http_proxy": {
```

```suggestion
        "httpProxy": {
```

The Feature options uses a camel case, can we switch to that?